pkg/utils/k8sutil.go

```diff
@@ -282,6 +282,16 @@ func CreateK8sResources(ns, name string, spec *spec.FunctionSpec, client *kubern
 									MountPath: "/kubeless",
 								},
 							},
+							LivenessProbe: &v1.Probe{
+								InitialDelaySeconds: int32(3),
+								PeriodSeconds:       int32(1),
```

Comment:
I suspect some of our runtimes might take a few seconds to get started up, so I expect we'll need to tune this over time.  I think what you really want here is to have a longer (default) livenessProbe, but shorter readinessProbe (in particular, we don't want to *kill* the container just because it wasn't ready in initialDelay+Period seconds).

Happy to merge the version you've got (in the next commit), and we can tune this further in followup PRs.